Branch-SDK/Branch-SDK/BNCContentDiscoveryManager.m

```diff
@@ -268,10 +472,10 @@ - (void)indexContentWithUrl:(NSString *)url spotlightIdentifier:(NSString *)spot
     void (^__nullable completionBlock)(NSError *indexError) = ^void(NSError *__nullable indexError) {
         if (callback) {
             if (indexError) {
-                callback(nil, indexError);
+                callback(nil, spotlightIdentifier, indexError);
             }
             else {
-                callback(url, nil);
+                callback(url, spotlightIdentifier, nil);
```

Comment:
same: need support for both types of callbacks, `callback` and `spotlightCallback`